scripts/nerfcapture2nerf.py

```diff
@@ -0,0 +1,254 @@
+
+from common import *
+import pyngp as ngp  # noqa
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--stream", action="store_true", help="Stream images directly to InstantNGP.")
+    parser.add_argument("--n_frames", default=10, type=int, help="Number of frames before saving the dataset. Also used as the number of cameras to remember when streaming.")
+    parser.add_argument("--save_path", required='--stream' not in sys.argv, type=str, help="Path to save the dataset.")
+    parser.add_argument("--depth_scale", default=4.0, type=float, help="Depth scale used when saving depth. Only used when saving dataset.")
+    parser.add_argument("--overwrite", action="store_true", help="Rewrite over dataset if it exists.")
+    return parser.parse_args()
+
+
+# DDS
+# ==================================================================================================
+@dataclass
+@annotate.final
+@annotate.autoid("sequential")
+class NeRFCaptureFrame(idl.IdlStruct, typename="NeRFCaptureData.NeRFCaptureFrame"):
+    id: types.uint32
+    annotate.key("id")
+    timestamp: types.float64
+    fl_x: types.float32
+    fl_y: types.float32
+    cx: types.float32
+    cy: types.float32
+    transform_matrix: types.array[types.float32, 16]
+    width: types.uint32
+    height: types.uint32
+    image: types.sequence[types.uint8]
+    has_depth: bool
+    depth_width: types.uint32
+    depth_height: types.uint32
+    depth_scale: types.float32
+    depth_image: types.sequence[types.uint8]
+
+
+dds_config = """<?xml version="1.0" encoding="UTF-8" ?> \
+<CycloneDDS xmlns="https://cdds.io/config" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://cdds.io/config https://raw.githubusercontent.com/eclipse-cyclonedds/cyclonedds/master/etc/cyclonedds.xsd"> \
+    <Domain id="any"> \
+        <Internal> \
+            <MinimumSocketReceiveBufferSize>10MB</MinimumSocketReceiveBufferSize> \
+        </Internal> \
+        <Tracing> \
+            <Verbosity>config</Verbosity> \
+            <OutputFile>stdout</OutputFile> \
+        </Tracing> \
+    </Domain> \
+</CycloneDDS> \
+"""
+# ==================================================================================================
+
+def set_frame(testbed, frame_idx: int, rgb: np.ndarray, depth: np.ndarray, depth_scale: float, X_WV: np.ndarray, fx: float, fy: float, cx: float, cy: float): 
+    testbed.nerf.training.set_image(frame_idx = frame_idx, img=rgb, depth_img=depth, depth_scale=depth_scale*testbed.nerf.training.dataset.scale)
+    testbed.nerf.training.set_camera_extrinsics(frame_idx=frame_idx, camera_to_world=X_WV)
+    testbed.nerf.training.set_camera_intrinsics(frame_idx=frame_idx, fx=fx, fy=fy, cx=cx, cy=cy)
+
+
+def live_streaming_loop(reader: DataReader, max_cameras: int):
+    # Start InstantNGP
+    testbed = ngp.Testbed(ngp.TestbedMode.Nerf)
+    testbed.init_window(640, 480)
+    testbed.reload_network_from_file(f"configs/nerf/base.json")
```

Comment:
Can you call `testbed.reload_network_from_file()` instead? `base.json` is the default and it will more robustly be found if you don't give it an explicitly relative path.